content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,146 @@ schema: olm.channel
+
+
+
+
+### Specifications
+The `composite template` is composed of two schemas that represent the operator author's role and the catalog maintainer's role.
+
+#### Operator Author
+Representing the operator author is the `olm.composite` schema which defines:
+- Where each input (`Component`) exists
+- How each contribution is mapped to a destination catalog (`Component.Name`). All destination catalogs defined in this file **must** also exist in the catalog configuration file created by the catalog maintainers
+- How each `Component` is processed to generate a catalog contribution (`Component.Strategy`)
+- The directory structure the catalog contribution generation should follow (`Component.Destination`)
+
+The cue schema for the `olm.composite` schema is:
+```cue
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+}
+```
+
+#### Catalog Maintainer
+Representing the catalog maintainer is the `olm.composite.catalogs` schema which defines:
+- One or more catalogs (`Catalog`) identified by a string (`Catalog.Name`)
+- Supported input formats (`Builder`) for each `Catalog` (`Catalog.Builders`) which is a list of strings where each item is a `Builder` schema. The currently supported `Builder` schemas are:
+    - `olm.builder.basic` which represents that the use of [basic templates](#basic-template) for catalog contribution generation is allowed
+    - `olm.builder.semver` which represents that the use of [semver templates](#semver-template) for catalog contribution generation is allowed
+    - `olm.builder.raw` which represents that the use of raw FBC for catalog contribution generation is allowed
+    - `olm.builder.custom` which represents that the use of custom catalog contribution generation logic is allowed
+- The expected directory structure of any generated contribution (`Catalog.Destination.WorkingDir`)
+
+The cue schema for the `olm.composite.catalogs` schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+### Examples
+**catalogs.yaml**
+
+The following example specifies a catalog configuration that defines a catalog named `v1` and the contribution generation logic to only accepting input that uses the [basic template](#basic-template). When using the `opm alpha render-template composite` subcommand the path to this file can be specified with the `-f` option and can be a file path or a URL (the URL must return the raw file contents).
```

Comment:
```suggestion
The following example specifies a catalog configuration that defines a catalog named `v1`. This example limits the input to the [basic template](#basic-template) for the contribution generation logic. If you use the `opm alpha render-template composite` subcommand, the path to this file can be specified with the `-f` option. The option can be a file path or a URL. If you use a URL, the URL must return the raw file contents.
```
Just shortening some sentences and trying to clarify a couple of points where I got confused.

Reply:
Addressed in https://github.com/operator-framework/olm-docs/pull/286/commits/1cdb364fe5c9f65cbc47ff7effdb3c41bf104cc0